templates/Pages/submission.php

```diff
@@ -1,21 +1,22 @@
 <?php $this->assign('title', 'Submission'); ?>
 
 <div class="submissions index content">
-    <h2>Submission</h2>
-
-    <div class="submissions-action">
-        <?php
-        echo $this->Html->link(_('Call for Submission'), [
-            'controller' => 'Pages',
-            'action' => 'display',
-            'cfs'
-        ], [
-            'class' => 'button'
-        ]);
-        ?>
-    </div>
-
-    <div class="both"></div>
+
+<div class="content">
+    <p class="call">
+        Submissions are not yet open for ISC23.
+    </p>
+
+    <h2>Call for Submissions</h2>
+
+    <p>
+        The IO500 is <b>no longer</b> accepting submissions for the
+        upcoming 11th semi-annual IO500 list in conjunction with SC'22.
```

Comment:
This shouldn't really say "upcoming" for the previous list.  Maybe "The IO500 \<b>will soon\</b> be accepting submissions for the upcoming 12th semi-annual IO500 list in conjunction with ISC'23."